client/lib/post-types-list/list.js

```diff
@@ -52,13 +54,13 @@ PostTypesList.prototype.fetch = function( siteId ) {
 
 		debug( 'getting PostTypesList from api' );
 		wpcom
+		.undocumented()
 		.site( siteId )
-		.postTypesList( function( error, data ) {
+		.postTypesList( { context: 'edit' }, function( error, data ) {
```

Comment:
`edit` is require to include `capabilities` the response [#](http://v2.wp-api.org/reference/types/)